src/api.js

```diff
@@ -9,6 +9,7 @@ window.api = {
         data: JSON.parse(data),
       });
     } catch (err) {
+      console.log(err);
```

Comment:
Definitely a good idea. If we use `console.error` instead of `console.log` it will do two things:

1. the text will show up in bright red.
2. easier to check for in tests, as all of our end-to-end tests fail when there's a log message with the 'error' type.